### PR TITLE
Improve text layout on mobile device #155

### LINKED ISSUE #155: Improve text layout on mobile device
For any smartphone whose view is 500px or below.

- Currently only about 2-4 words for bullet points

- Table text shown in excessively large font-size

### PR DESCRIPTION
**Pull request by acjh**
Fixes #155
<a href="http://acjh.github.io/website">Preview</a>

| Before | After |
| -------- |--------|
|  |  |
|  |  |
|  |  |
|  |  |

### CODE REVIEW

File: styles/common.css [hunk 1]
```diff
@@ -434,6 +434,33 @@ img {
...
+        padding-right: 10px !important;
+    }
+
+    ul {
+        padding-left: 20px !important;
+    }
+
+    table.bordered tr > td {
+        font-size: 12px !important;
+    }
```
Comment (wkurniawan07) [inline styles/common.css]: Actually, the oversized table text size is an oversight on my part when I did #130. The font size should be 14px which is the same as the others. I'll have it fixed right up so you can remove this definition, or is there another reason to make it even smaller?

**Comment by wkurniawan07** [comment]
Good work :+1: just that one minor issue.

File: styles/common.css (hunk 1 above)
Comment (acjh) [inline styles/common.css]: 12px for table text seems better than 14px on mobile, as it is still large enough but easier to scan for information, as well as reduces scrolling. Limited observation though, as the example above appears to be the only instance of that class.

Comment (wkurniawan07) [inline styles/common.css]: Ok, we can let this in for now, later on when I fix up my mistake in #130 I'll see which alternative is better. Good job on this :+1:

Comment (acjh) [inline styles/common.css]: Thanks!

**Pull request merged**